testsuite/features/step_definitions/command_steps.rb

```diff
@@ -66,7 +67,8 @@
     # TODO: move that internal resource to some other external location
     STDERR.puts 'Sanity check not implemented, move resource to external network first'
```

Comment:
Same remark: isn't that done now? If so, please remove this error message (the whole "Uyuni" case).

Reply:
resolved